### DISCUSSION TITLE
[Bug]: the instructions on the Get Started site do not work.

### DISCUSSION THREAD

**Discussion by joshua-lockhart** [django-wiki/django-wiki#1368] [Bug]
### Describe the bug

I am following the instructions here: https://django-wiki.readthedocs.io/en/releases-0.7.6/development/index.html#get-started

Two issues emerge:
1. when I run the testproject the css is not loading
2. when I run pytest, I get a wall of failed tests

### Steps to reproduce.

ISSUE 1.
1. follow instructions in 
https://django-wiki.readthedocs.io/en/releases-0.7.6/development/index.html#get-started to set up virtual env etc
2. cd [...]/django-wiki/testproject
3. python manage.py runserver
4. browse to site, observe no CSS

ISSUE 2
1. follow step 1 above
2. run pytest
3. observe wall of failed tests

I see "django.core.exceptions.ImproperlyConfigured: Requested setting INSTALLED_APPS, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings."
a lot. Presumably I have not configured something correctly, the DJANGO_SETTINGS_MODULE no doubt. However, it is not clear to me what this should be set to to run the tests.

### Python Version

3.12.3

### Django Version

5.0.8

**Comment by benjaoming** [comment]
Hi @joshua-lockhart - thanks for giving this feedback! This is likely because somehow the pysassc command (it's run when building the project with Hatch) hasn't built the project yet. Maybe there is something missing in the docs @oscarmcm ? I can have a look tomorrow maybe :+1:

**Reply by joshua-lockhart** [reply]
ah what is this hatch, this could be the missing link

**Reply by joshua-lockhart** [reply]
ok got hatch running. 

1. hatch env create works fine, NOT like what is described here (https://github.com/django-wiki/django-wiki/discussions/1366)
2. but when I do hatch run test:all it throws up

ModuleNotFoundError: No module named 'tests'

and now I can't even run the test project, it keeps repeating that same error.

**Reply by benjaoming** [reply]
That is odd - do you see the `tests` folder (which is a Python module) in the folder you are running the commands from?

**Reply by joshua-lockhart** [reply]
yeah it's there!

**Reply by joshua-lockhart** [reply]
"hatch run test" seems to be running OK

**Reply by benjaoming** [reply]
`hatch run test:all` works for me :thinking:

**Reply by joshua-lockhart** [reply]
OK. It's frustrating because now I can't even run the testproject site with manage.py, it's still complaining about not finding the tests module

**Reply by benjaoming** [reply]
Can you try this from a clean directory? This works for me:

```
git clone git@github.com:django-wiki/django-wiki.git
cd django-wiki
hatch env create
cd testproject
python manage.py migrate
python manage.py runserver
```
